src/lib/declarations.ts

```diff
@@ -290,6 +290,16 @@ export const EXCLUDED_KEY_BINDS = [
   "AltRight",
 ];
 
+/**
+ * The set of modifers that will be checked to determine if a given key should
+ * be ignored when handling key events.
+ *
+ * Note that these correspond to `KeyboardEvent.key` values since `KeyboardEvent.getModifierState`
+ * does not accept `KeyboardEvent.code` values.
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState}
+ */
+export const EXCLUDED_KEY_MODIFIERS = ["Alt", "Control", "Meta", "Shift"];
```

Comment:
I believe these modifiers are safe to ignore (i.e. ignoring them won't interfere with shorctuts) since they're all included in the set of keys that can't be used for key binds.